lib/db-layer.js

```diff
@@ -4,12 +4,18 @@ module.exports.replace = () => ({
   'r.pool.run(r.table(__a).insert(__b))': 'r[__a].insert(r, __b)',
```

Comment:
Some more candidates for auto fixing?

`r.pool.run(r.table('courseGroups').get(course.courseGroup)('asset').default(null))` => `r.courseGroups.getAttributesById(r, [course.courseGroup], ['asset'])?.[0]?.asset` (would require adding `getAttributesById` to all repositories, which I think is a smart move anyway)

`r.pool.run(r.table('layouts').get(client.dashboardLayout).delete())` => `r.layouts.destroyById(r, client.dashboardLayout)` (would require adding `destroyById` to all repositories. This one is a little more debatable as some tables don't delete things in this way, e.g. soft deletions)